src/assets/locales/en/translation.json

```diff
@@ -636,5 +637,76 @@
   "earn_strategy": {
     "withdraw_rewards_in_wrapped": "Withdraw rewards in {{wrappedCurrencySymbol}}:",
     "update_position": "Update position"
+  },
+  "transaction": {
+    "confirm_transaction_wallet": "Confirm this transaction in your wallet",
+    "confirm_transaction": "Confirm transaction",
+    "transaction_processing": "Transaction processing",
+    "transaction_failed": "Transaction failed",
+    "transaction_successful": "Transaction successful",
+    "swapping_to": "Swapping {{fromAmount}} {{fromCurrency}} to {{toAmount}} {{toCurrency}}",
+    "swapped_to": "Swapped {{fromAmount}} {{fromCurrency}} to {{toAmount}} {{toCurrency}}",
+    "adding_liquidity_to_pool": "Adding liquidity to {{poolName}} Pool",
+    "added_liquidity_to_pool": "Added liquidity to {{poolName}} Pool",
+    "removing_liquidity_from_pool": "Removing liquidity from {{poolName}} Pool",
+    "removed_liquidity_from_pool": "Removed liquidity from {{poolName}} Pool",
+    "claiming_pool_rewards": "Claiming pools rewards",
```

Comment:
For the pool rewards, would be nice to add the amount and currency as well

Reply:
That might create a very big description of the transaction. Description with more than 2 lines of text, the rest of the text gets hidden. I suggest we show the amount in dollars.

Reply:
Yeah, good suggestion

Reply:
Just red my response...sorry for my poor english ahaha in my head made sense. What I meant is that we want to avoid having very big notifications that might overlap the UI a lot, so we limitted to 2 lines of description, where there rest of the text gets hidden. You anwsered my question, but I just wanted to make it cleared :D